### PR TITLE
Add documentation for Pulsar OAuth configuration.

### PR DESCRIPTION
**Pull request by JeffBolle**
Documentation for configuring Pulsar ingest plugin for OAuth2 authentication. Supports feature PR here: apache/pinot#12068

### CODE REVIEW

File: basics/data-import/pinot-stream-ingestion/apache-pulsar.md [hunk 1]
```diff
@@ -63,12 +63,22 @@ You can change the following Pulsar specifc configurations for your tables
 
 ### Authentication
 
-The Pinot-Pulsar connector supports authentication using the security tokens. You can generate the token by following the [official Pulsar documentaton](https://pulsar.apache.org/docs/en/security-token-client/). Once generated, you can add the following property to `streamConfigs` to add auth token for each request
+The Pinot-Pulsar connector supports authentication using the security tokens. You can generate the token by following the [official Pulsar documentation](https://pulsar.apache.org/docs/en/security-token-client/). Once generated, you can add the following property to `streamConfigs` to add auth token for each request
```
Comment (kelseiv) [inline basics/data-import/pinot-stream-ingestion/apache-pulsar.md]:
```suggestion
The Pinot-Pulsar connector supports authentication using security tokens. To generate a token, follow the instructions in [Pulsar documentation](https://pulsar.apache.org/docs). Once generated, add the following property to `streamConfigs` to add an authentication token for each request:
```

Comment (kelseiv) [reply basics/data-import/pinot-stream-ingestion/apache-pulsar.md]: appears the link is no longer working, added link to top-level docs site

File: basics/data-import/pinot-stream-ingestion/apache-pulsar.md [hunk 2]
```diff
@@ -63,12 +63,22 @@ You can change the following Pulsar specifc configurations for your tables
...
 
 ```
 "stream.pulsar.authenticationToken":"your-auth-token"
 ```
 
+### OAuth2 Authentication
+
+The Pinot-Pulsar connector support authentication using OAuth2, for example if connecting to a StreamNative Pulsar cluster. This is covered in the [official Pulsar documentation](https://pulsar.apache.org/docs/en/security-oauth2/#configure-oauth2-authentication-in-pulsar-clients). Once configured, you can add the following properties to `streamConfigs`:
```
Comment (kelseiv) [inline basics/data-import/pinot-stream-ingestion/apache-pulsar.md]:
```suggestion
The Pinot-Pulsar connector supports authentication using OAuth2, for example, if connecting to a StreamNative Pulsar cluster. For more information, see how to [Configure OAuth2 authentication in Pulsar clients](https://pulsar.apache.org/docs/en/security-oauth2/#configure-oauth2-authentication-in-pulsar-clients). Once configured, you can add the following properties to `streamConfigs`:
```

**Review by kelseiv — CHANGES_REQUESTED** [review]

**Comment by kelseiv** [comment]
thank you, @JeffBolle!

File: basics/data-import/pinot-stream-ingestion/apache-pulsar.md (hunk 1 above)
Comment (JeffBolle) [reply basics/data-import/pinot-stream-ingestion/apache-pulsar.md]: Sure. I was just correcting a typo there. I have found the updated link to the JWT token auth page.

**Comment by JeffBolle** [comment]
@kelseiv Sure thing. Happy to. Now I just need to get someone to review the actual code PR.

**Review by kelseiv — APPROVED** [review]

**Pull request merged**